transcription/utils/image_utils.py

```
@@ -0,0 +1,85 @@
+    # key: occurrence_id 
+    # value: ground truth taxon, date, or geography 
+    # returns a dictionary 
+    data_dict = {}
+    with open(gt_txt_file, "r") as f: 
+        lines = f.readlines()
+        for line in lines:
+            try:
+                pattern = r"(?<=\d):"  # Match ':' only if preceded by a digit
+                key, value = re.split(pattern, line.strip())
+                data_dict[key] = value
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)